app/src/main/java/com/example/caregiver/ProfileInfo.java

```diff
@@ -251,8 +269,18 @@ public void onClick(View v) {
         }
     };
 
-    public static class MainActivity extends AppCompatActivity {
+    /**
+     * Function to logout
+     */
+    private View.OnClickListener logOutListener = new View.OnClickListener() {
+        @Override
+        public void onClick(View v) {
+            Intent i = new Intent(v.getContext(), Login.class);
+            startActivity(i);
+        }
```

Comment:
I wonder if pressing the back button after logging out, will take the user in the app again. Were you able to test this?

Reply:
LOL you are right. It goes back to the app. I add in 
```java
getActivity().finish()
```
This should prevent it to go back.